specifyweb/frontend/js_src/lib/components/Forms/Save.tsx

```diff
@@ -34,6 +34,21 @@ import { FORBID_ADDING, NO_CLONE } from './ResourceView';
 
 export const saveFormUnloadProtect = formsText.unsavedFormUnloadProtect();
 
+const resourcesToNotClone = new Set([
```

Comment:
don't hardcode this list
instead generate it on the fly like so:
get a list of app-resources in this list that have a "name" field set to some string: https://github.com/specify/specify7/blob/3d3305bd68671e4841e3fa77c9db3c438d1615f1/specifyweb/frontend/js_src/lib/components/AppResources/types.tsx#L70